lib/Backend/BackwardPass.cpp

```diff
@@ -1818,6 +1908,15 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr)
             BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed = byteCodeUsesInstr->byteCodeUpwardExposedUsed;
             if (byteCodeUpwardExposedUsed != nullptr)
             {
+                //Stack Args for Formals optimization.
```

Comment:
So it is guaranteed not to be used by another instr?  A better implementation would have been to set the bit because of this instr in the first place.  Now you are turning it off even if someone else had a valid use.

---
In reply to: [67800813](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67800813) [](ancestors = 67800813,67740116)